.github/workflows/end2end.yaml

```
@@ -0,0 +1,406 @@
+---
+name: build-iso-and-end2end-test
+
+concurrency:
+  group: "zenko-tests"
```

You only want one build at a time? Or one build per branch?
```suggestion
  group: "zenko-tests-${{ github.ref_name }}"
```

---

Yes you are right, I updated that, thanks!